### PR TITLE
Adds Remote Adding/Pinning and associated new methods/APIs

### PR DESCRIPTION
**Pull request by carsonfarmer**

### CODE REVIEW

File: mobile/node.go [hunk 1]
```diff
@@ -171,7 +170,29 @@ func (n *Node) Stop() error {
 	return nil
 }
 
-func (n *Node) PinPhoto(path string, thumb string) (string, error) {
+func (n *Node) AddPhotoRemote(path string, thumb string) ([]wallet.Hashed, error) {
```
Comment (asutula) [inline mobile/node.go]: Huh I don't see these changes reflected in the `Mobile.framework` header... Do we still need to re-generate `Mobile.framework`?

Comment (carsonfarmer) [reply mobile/node.go]: Hah, I haven't updated it yet :) Hence the WIP. But also, I'm going to change it 'back' a little bit. So there will be _minimal_ changes to the actual Mobile.framework header and associated wrapper stuff...

**Comment by carsonfarmer** [comment]
that should now be updated with the new framework now @asutula. i haven't gone into textile-photos to update the wrapper stuff there yet though

**Comment by asutula** [comment]
Cool, I can make updates to the app based on the new framework.

**Comment by carsonfarmer** [comment]
saweet! so just let me know if that's enough information from the remote server aaron. right now, i'm mostly relying on errors to propagate up... but if you need header information for some reason, let me know and i can probably return that stuff as well

**Pull request merged**